units/XNL0402/XNL0402_Script.lua

```diff
@@ -160,41 +175,48 @@ XNL0402 = Class(NLandUnit) {
 
     --wait time needed to allow the beamer to retarget without the fake beam being turned on and off all the time
     --currently this relies on completing everything instantly after the wait time. if this changes, more thinking is needed
+    ---@param self XNL0402
     BeamEffectsDisableThread = function(self)
         WaitTicks(20)
         self.BeamHelperFxBag:Destroy()
         self.BeamChargeUpFxBag:Destroy()
         if self.Beaming then
             for k, v in NomadsEffectTemplate.PhaseRayFakeBeamMuzzleBeamingStopped do
-                emit = CreateAttachedEmitterColoured( self, 'ReactorBeam01', self.Army, v )--:OffsetEmitter(0, 0.1, 0) --fading light
+                emit = CreateAttachedEmitterColoured( self, 'ReactorBeam01', self.Army, v )
```

Comment:
```suggestion
                emit = CreateAttachedEmitterColoured(self, 'ReactorBeam01', self.Army, v)
```

Reply:
Also visual FX can be hard to see so they should be annotated somewhere. Either a comment here saying that this is a fading light, or in the effect template if it's a single emitter blueprint in the template (otherwise describe every emitter bp in the template individually if there are multiple).